WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -376,20 +376,27 @@ private ConstructionState AddBlameScript(Round round, ConstructionState coinjoin
 		return coinjoin;
 	}
 
-	private ConstructionState AddCoordinatorFee(Round round, ConstructionState coinjoin)
+	private ConstructionState AddCoordinationFee(Round round, ConstructionState coinjoin)
 	{
 		Script coordinatorScriptPubKey = GetCoordinatorScriptPreventReuse(round);
 
 		var coordinationFee = round.Alices.Where(a => !a.IsPayingZeroCoordinationFee).Sum(x => round.CoordinationFeeRate.GetFee(x.Coin.Amount));
-		coordinationFee -= round.FeeRate.GetFee(coordinatorScriptPubKey.EstimateOutputVsize());
-
-		if (coordinationFee > coinjoin.Parameters.AllowedOutputAmounts.Min)
+		if (coordinationFee == 0)
 		{
-			coinjoin = coinjoin.AddOutput(new TxOut(coordinationFee, coordinatorScriptPubKey));
+			round.LogInfo($"Coordination fee wasn't taken, because it was free for everyone. Hurray!");
 		}
 		else
 		{
-			round.LogWarning($"Coordinator fee wasn't taken, because it was too small: {nameof(coordinationFee)}: {coordinationFee}.");
+			coordinationFee -= round.FeeRate.GetFee(coordinatorScriptPubKey.EstimateOutputVsize());
+
+			if (coordinationFee > coinjoin.Parameters.AllowedOutputAmounts.Min)
+			{
+				coinjoin = coinjoin.AddOutput(new TxOut(coordinationFee, coordinatorScriptPubKey));
+			}
+			else
+			{
+				round.LogWarning($"Coordination fee wasn't taken, because it was too small: {nameof(coordinationFee)}: {coordinationFee}.");
```

Comment:
Imo this creates confusion, or at least for me, because it will say something like:

> Coordination fee wasn't taken, because it was too small: -71

Or, 

> Coordination fee wasn't taken, because it was too small: 12

But the coordination fees were not -71 or 12 neither. The coordination fees were 54 and 90.

For this reason I would log the real coordination fee and the resulting output value after fee. Something like:

> Coordination fee wasn't taken, because it was too small, 54sats, resulting in net output -71 after mining fees.

Reply:
I introduced the `effectiveCoordinationFee` variable. IMO that makes sense to see in the logs.